app.rb

```diff
@@ -1,9 +1,37 @@
 require 'sinatra/base'
+require_relative './lib/message_handler'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
```

Comment:
It's always good to remember to remove test code before submitting code for review (anywhere, not just at Makers)